### PR TITLE
Remove condition from yaml env variables section

### PR DESCRIPTION
**Pull request by abhipsaMisra**
Fix the following error: 

image

### CODE REVIEW

**Review by drwill-ms — APPROVED** [review]

File: vsts/vsts.yaml [hunk 1]
```diff
@@ -340,8 +340,6 @@ jobs:
           DPS_SYMMETRIC_KEY_INDIVIDUAL_ENROLLMENT_REGISTRATION_ID: $(DPS-SYMMETRIC-KEY-INDIVIDUAL-ENROLLMENT-REGISTRATION-ID)
           DPS_SYMMETRIC_KEY_INDIVIDUAL_ENROLLEMNT_PRIMARY_KEY: $(DPS-SYMMETRIC-KEY-INDIVIDUAL-ENROLLEMNT-PRIMARY-KEY)
 
-        condition: always()
```
Comment (abhipsaMisra) [inline vsts/vsts.yaml:343]: Conditions are generally used in context of tasks/jobs.

**Review by schoims — APPROVED** [review]

**Review by patilsnr — APPROVED** [review]

**Comment by patilsnr** [comment]
It's interesting that `gatedBuild.ps1` is displaying a build error despite not being part of this PR

**Comment by abhipsaMisra** [comment]
> It's interesting that `gatedBuild.ps1` is displaying a build error despite not being part of this PR

The error is getting flagged irrespective of a PR actually. Just go over to DevOps and pull up the west central US pipeline and you should see the error pop up at the top.

**Review by drwill-ms — APPROVED** [review]

**Pull request merged**